src/routes/Random.tsx

```diff
@@ -0,0 +1,11 @@
+import Header from '../components/Header/Header';
+import RandomImage from '../components/RandomImage/RandomImage';
+
+export default function Random() {
+    return(
+        <>
+            <Header />
```

Comment:
This component is reused on all your route level components. Is there a way to avoid this ? What do you think ?

Reply:
> Hi Kevin! Thank you for the time and effort you put in the project. I have a couple of questions if you don't mind answering:
> 
>     * Why did you use Typescript over plain JS ?

Thank you a lot for your feedback Dimitris! I used TypeScript because it was specified on the challenge prompt. I don't have as much experience using TS as I have using simple JS, but I enjoy learning new technologies and thought this was a great opportunity to expand my TypeScript knowledge.

From the challenge prompt (https://github.com/GlobalWebIndex/platform2.0-react-challenge-2): 
> We give you a lot of freedom in technologies and ways of doing things. We only insist on you using React.js with Typescript.

>     * When going back and forth on the  `/breeds` route, the component fetches the data from the api again. Do you know any way / tool to avoid recalling the api ?

This is an oversight on my end. I planned to use sessionStorage (or even localStorage, in case I'm 100% sure no breeds will be added and the formatting of the api response won't change), as I had done in the Breed.tsx component. I **think** Redux is a popular tool that can also be used for this, but I have never used it and thought it was a very large library for just a couple of api calls. I might be wrong though!

> This component is reused on all your route level components. Is there a way to avoid this ? What do you think ?

I should have created a Layout component with Header inside it to wrap all my routes, like this:
`<Route path="/" element={<Random />} />
        <Route path="breeds" element={<Breeds />} />
        <Route path="breeds/:breed" element={<BreedGallery />} />
        <Route path="*" element={<NoMatch />} />`